locale/hi/LC_MESSAGES/loris.po

```diff
@@ -201,3 +213,74 @@ msgstr "पासवर्ड पर्याप्त जटिल नहीं
 
 msgid "This password is known to be exposed in online data breaches."
 msgstr "यह पासवर्ड ऑनलाइन डेटा उल्लंघनों में उजागर होने के लिए जाना जाता है।"
+
+msgid "Data Supervisors to Email"
+msgstr "ईमेल करने के लिए डेटा पर्यवेक्षक"
+
+msgid "Invalid email address"
+msgstr "अमान्य ईमेल पता"
+
+# Common strings on widgets
+msgid "NEW"
+msgstr "नया"
+
+msgid "Updated"
+msgstr "अद्यतन किया गया"
+
+msgid "Uploaded"
+msgstr "अपलोड किया गया"
+
+# User Account related
+msgid "Password Rules"
+msgstr "पासवर्ड नियम"
+
+msgid "Username"
+msgstr "उपयोगकर्ता नाम"
+
+msgid "User name"
+msgstr "उपयोगकर्ता नाम"
```

Comment:
I don't think we need both, let's just keep **Username** and change for this key wherever we have `User name`

Reply:
@SKADE2303 I consulted @driusan in this one and he agreed it should be `Username` only. You have green light for making the respective changes. Thanks !!